tools/code_format/config.yaml

```diff
@@ -185,6 +185,7 @@ paths:
     - source/extensions/matching
     - source/extensions/network
     - source/extensions/path
+    - source/extensions/quic/server_preferred_address
```

Comment:
Note that the existing fixed server preferred address code was already calling `ProtoExceptionUtil::throwProtoValidationException`, so this isn't adding a new set of exceptions; this is just more of the same in the same extension type, and it is unlikely to be needed on mobile.